migrations/versions/ecad721e2a38_rename_committee_to_directorship.py

```diff
@@ -0,0 +1,30 @@
+"""rename committee to directorship
+
+Revision ID: ecad721e2a38
+Revises: 3eaae92ce6b3
+Create Date: 2022-03-27 15:12:33.584620
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'ecad721e2a38'
+down_revision = '3eaae92ce6b3'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.rename_table('freshman_committee_attendance', 'freshman_directorship_attendance')
+    op.rename_table('committee_meetings', 'directorship_meetings')
+    op.rename_table('member_committee_attendance', 'member_directorship_attendance')
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.rename_table('freshman_directorship_attendance', 'freshman_committee_attendance')
+    op.rename_table('directorship_meetings', 'committee_meetings')
+    op.rename_table('member_directorship_attendance', 'member_committee_attendance')
+    # ### end Alembic commands ###
```

Comment:
notably, it looks like this doesn't rename the enum? or any of the column names?